docs/source/users/problem_definition_files/native.rst

```diff
@@ -75,6 +75,10 @@ function
   Inside FitBenchmarking, this is passed on to the specified software and, as
   such, the format is specific to the package we wish to use, as described below.
 
+plot_scale
```

Comment:
The below **IVP**, **Mantid**,... is still part of the funtion section above, so this should probably be below or above all of that.

Can you change default to linear in the options as well? We've had a lot of confusion with calling default options "default" in the config file.

Also a very minor point but I'd say x and y rather than y and x.